osu.Game/Screens/Play/ReplaySettings/CollectionSettings.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) 2007-2017 ppy Pty Ltd <contact@ppy.sh>.
+// Licensed under the MIT Licence - https://raw.githubusercontent.com/ppy/osu/master/LICENCE
+
+using osu.Framework.Allocation;
+using osu.Framework.Graphics;
+using osu.Game.Graphics.Sprites;
+using osu.Game.Overlays.Music;
+using System.Collections.Generic;
+using osu.Game.Graphics.UserInterface;
+
+namespace osu.Game.Screens.Play.ReplaySettings
+{
+    public class CollectionSettings : SettingsDropdownContainer
+    {
+        protected override string Title => @"collections";
+
+        [BackgroundDependencyLoader]
+        private void load()
+        {
+            Children = new Drawable[]
+            {
+                new OsuSpriteText
+                {
+                    Text = @"Add current song to",
+                },
+                new CollectionsDropdown<PlaylistCollection>

```

Comment:
We'll need `CollectionsDropdown : SettingsDropdown` to provide the label for us, otherwise the label and the dropdown are too far apart from one another. Check how I handled  this with ReplaySettingsCheckbox/ReplaySettingsSliderBar.